boot/bootutil/src/loader.c

```diff
@@ -185,11 +201,17 @@ boot_status_source(void)
              table->bst_magic_scratch   == state_scratch.magic) &&
             (table->bst_copy_done_slot0 == 0    ||
              table->bst_copy_done_slot0 == state_slot0.copy_done)) {
-
-            return table->bst_status_source;
+            uint8_t source = table->bst_status_source;
```

Comment:
emacs sounds great, let's use Mynewt style, as the majority of the code is written in that format already.  Let's bring the coding style document into the mcuboot project.  Unless there are any major objections?  My only sticking point is the avoidance of tabs.